### PR TITLE
Add statement to explain responding to invite

### PR DESCRIPTION
**Pull request by evansalz**
Added a new print statement saying:

System.out.println("To respond to this invite, enter 2.\n");

### CODE REVIEW

File: src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java [hunk 1]
```diff
@@ -179,6 +179,7 @@ public void OnEvent(Event e, TCPConnection connect) throws IOException {
 			String invFrom = new String(inF);
 			gamesInvitedTo.add(invFrom);
 			System.out.println("\nYou received an invite from " + invFrom + "!");
+			System.out.println("To respond to this invite, enter 2.\n");
```
Comment (FoxXix) [inline src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: This only works if the user is on the main menu to press 2. They can be anywhere else and get this notification and '2' will not work for the other menu they are on.

Comment (evansalz) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: Well, then what do they use in that case?

Comment (evansalz) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: Is it exit and then 2?

Comment (FoxXix) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: They go to the manage invites screen to accept the invite

Comment (evansalz) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: I know that.  You enter 2 to get to the manage invites screen.  If you are not on the main menu, you enter 'exit' then it takes you back to the main menu and then you can enter 2 to manage invites.

Comment (evansalz) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: How about this?

	System.out.println("To respond to this invite, enter '2'.\n");
	System.out.println("If you are not on the main menu, enter 'exit' and then '2'.\n");

Comment (FoxXix) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: It can be exit and then 2. Unless they are in the menu to view user profiles.

Comment (evansalz) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: Exit and 2 works the same for me on that menu too.  Is this the menu you're talking about?
```

scoobs's Profile

Joined: 2018/10/27 17:45:45
Wins: 0
Losses: 0
Draws: 0
Forfeits: 0

1) Search for player
To exit, type 'exit' and press Enter
```

Comment (FoxXix) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: One more deeper. Press one on that screen.

Comment (evansalz) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: Well, these statements are not a perfect solution, but most of the time a user will not be mid-search for a second profile when they receive a notification.  So I feel like this covers most of the cases.

Comment (evansalz) [reply src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java]: I've updated it to have the 2 and the exit, 2 options.

**Pull request merged**